### PR TITLE
Add react strict dom

### PR DESCRIPTION
**Pull request by dgca**
Swaps out React Native Web for `react-strict-dom` and adds a simple button example

### CODE REVIEW

File: packages/tackle-box/.storybook/main.ts
Comment (cgjohn) [inline packages/tackle-box/.storybook/main.ts:1]: Do we think we will want to bring back storybook as we get things setup?

**Review by cgjohn — APPROVED** [review]
Everything look good! Only real question right now is if we need to keep anything around in the `src` file or curious what your thoughts are around previewing components as we work on them.

File: packages/tackle-box/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
```
[the remark this answers is not in the document]
Comment (dgca) [reply packages/tackle-box/README.md:1]: Yeah, def. We should probably just start adding these things as backlog items to the project. I'll add one for Tackle Box README

**Comment by dgca** [comment]
@cgjohn R.e. src, let's keep it for now cause it might help us debug the web build when we set up Storybook.

R.e. previewing, we can add them to the iOS app for now in the ui.tsx route. Def should come up with a better plan _soon_ but I'll just merge this as is and we can come up with a better solution as we build out the components.

**Pull request merged**